keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -691,11 +755,18 @@ defined in the k8s.io/apis/core/v1 package. Thus, every addition of a new
 condition type will require an API review. The constants will allow users of the
 package to reduce the risk of typing mistakes.
 
-Additionally, for Beta, we will re-evaluate an idea of a generic opinionated condition
-type indicating that a pod can be retried, for example `DisruptionTarget`.
+Additionally, we introduce a pair of generic condition types:
+- `DisruptionTarget`: indicates pod disruption (due to e.g. preemption,
+API-initiated eviction or taint-based eviction).
+- `ResourceLimitExceeded`: indicates the pod is evicted due to exceeding
+one of it's configured limits (e.g. on memory or ephemeral-storage).
+
+The more detailed information about the reason of disruption or the exceeded
+resource limit is conveyed by the `reason` and `message` fields of the added pod
+condition.
```

Comment:
IIUC, it seems that users can only define policy based on the condition type only and not also based on the reason, if true, please mention that explicitly.

Reply:
Indeed, matching by `reason` is not supported currently. I have added a paragraph in the section to mention it explicitly. Please take a look.

Reply:
What kind of "more detailed information"? You can say the actor that initiated it or the resource type that is exceeded.

Reply:
Updated